mapr/metadata.csv

```diff
@@ -0,0 +1,158 @@
+mapr.cldb.cluster_cpubusy_percent,gauge,10,percent,,The aggregate percentage of busy CPUs in the cluster.,0,mapr,
+mapr.cldb.cluster_disk_capacity,gauge,10,gibibyte,,The storage capacity for MapR disks in GB.,0,mapr,
+mapr.cldb.cluster_diskspace_used,gauge,10,gibibyte,,The amount of MapR disks used in GB.,-1,mapr,
+mapr.cldb.cluster_memory_capacity,gauge,10,mebibyte,,The memory capacity in MB.,0,mapr,
+mapr.cldb.cluster_memory_used,gauge,10,mebibyte,,The amount of used memory in MB.,-1,mapr,
+mapr.cldb.containers,gauge,10,container,,The number of containers currently in the cluster.,0,mapr,
+mapr.cldb.containers_created,count,10,container,,The cumulative number of containers created in the cluster. This value includes containers that have been deleted.,0,mapr,
+mapr.cldb.containers_unusable,gauge,10,container,,The number of containers that are no longer usable. The CLDB marks a container as unusable when the node that stores the container is offline for 1 hour or more.,-1,mapr,
+mapr.cldb.disk_space_available,gauge,10,gibibyte,,The amount of disk space available in GB.,1,mapr,
+mapr.cldb.nodes_in_cluster,gauge,10,node,,The number of nodes in the cluster.,0,mapr,
+mapr.cldb.nodes_offline,gauge,10,node,,The number of nodes in the cluster that are offline.,-1,mapr,
+mapr.cldb.rpc_received,count,10,operation,,The number of RPCs received.,0,mapr,
+mapr.cldb.rpcs_failed,count,10,operation,,The number of RPCs failed.,-1,mapr,
+mapr.cldb.storage_pools_cluster,gauge,10,,,The number of storage pools.,0,mapr,
+mapr.cldb.storage_pools_offline,gauge,10,,,The number of offline storage pools.,-1,mapr,
+mapr.cldb.volumes,gauge,10,volume,,"The number of volumes created, including system volumes.",0,mapr,
+mapr.db.flushes,count,10,flush,,The number of flushes that reorganize data from bucket files (unsorted data) to spill files (sorted data) when the bucket size exceeds a threshold.,0,mapr,
+mapr.db.forceflushes,count,10,flush,,The number of flushes that reorganize data from bucket files (unsorted data) to spill files (sorted data) when the in-memory bucket file cache fills up.,0,mapr,
+mapr.db.get_rpcs,count,10,operation,,The number of MapR database get RPCs completed,0,mapr,
+mapr.db.get_bytes,count,10,byte,,The number of bytes read by get RPCs,0,mapr,
+mapr.db.put_bytes,count,10,byte,,The number of bytes written by put RPCs,0,mapr,
+mapr.db.scan_rpcs,count,10,operation,,The number of MapR Database scan RPCs completed,0,mapr,
+mapr.db.scan_bytes,count,10,byte,,The number of bytes read by scan RPCs,0,mapr,
+mapr.db.append_rpcs,count,10,operation,,The number of MapR Database append RPCs completed,0,mapr,
+mapr.db.get_currpcs,gauge,10,operation,,The number of MapR Database get RPCs in progress,0,mapr,
+mapr.db.put_currpcs,gauge,10,operation,,The number of MapR Database put RPCs in progress,0,mapr,
+mapr.db.put_rpcs,count,10,operation,,The number of MapR Database put RPCs completed,0,mapr,
```

Comment:
The first two one are the number of operation "in progress", thus we probably want to keep them as a gauge (aka don't sum them when zooming out in a graph)

The last one is the number of operations during those 10 seconds of the interval, this we likely want to keep as a count so that when zooming out, each datapoint is the total number of operations during the interval.